data/lang/equipment-core/en.json

```diff
@@ -307,6 +307,10 @@
     "description": "",
     "message": "Guided Missile"
   },
+  "MISSILE_GUIDED_DESCRIPTION": {
+    "description": "Equipment description",
+    "message": "A guided missile able to home in on your target using advanced LIDAR."
```

Comment:
"home" -> "hone"

Reply:
"home in" -> target or approach a destination
"hone" -> sharpen, eg, a knife..

Reply:
*sigh*  .. although it appears that, as usual, there is some language-mangling going on across the Pond and "hone in" is starting to see accepted usage over there.  (and yes, I will brutally mock Americans for their corruption of the English language; not that the British can lay any claims to their sophistication of spelling their language in the first place; I mean, Worcestershiresauce (woostersheersoz) anyone???)

Reply:
Please tell me that you do do not not subscribe to the Oxford Comma? There is absolutely no way for a computer programmer to make logical sense of a list which does not use it.. apart from that I have no idea what the difference between the two is.

Reply:
Obviously I prefer British English 😁 go for "home in" 👍

Reply:
"Home" / "Homing" is the correct verb to use. In this specific technical context, "hone" is entirely incorrect, as the process refers to guiding the missile to the source of some emission - i.e. Home-On-Jam.